update-config-files.php

```diff
@@ -0,0 +1,361 @@
+                        $current_var[VAR_VALUE] .= $token;
+                    }
+                }
+                elseif ($token === ';')
+                {
+                    if ($current_var[VAR_NAME] && $current_var[VAR_VALUE] !== false)
+                    {
+                        $current_var[VAR_VALUE] = ltrim($current_var[VAR_VALUE]);
+                        $vars[$current_var[VAR_NAME]] = $current_var;
+                    }
+                    $current_var = array(VAR_NAME => false,
+                                        VAR_VALUE => false,
+                                        VAR_COMMENT => '', );
+                }
+                elseif ($current_var[VAR_VALUE] !== false)
+                {
+                    $current_var[VAR_VALUE] .= $token;
+                }
+            }
+        }
+
+        return $vars;
+    }
+
+    /**
+     * [formatComments formatComments adds slashes]
+     * @param  [string] $comments [comments in the file]
+     * @return [string] [returns comment with slashes]
+     */
+    function formatComments($comments)
+    {
+        if (empty($comments))
+        {
+            return '';
+        }
+        $lines = explode("\n", $comments);
+        $output = '';
+        foreach ($lines as $line)
+        {
+            $line = trim($line);
+            if (empty($line))
+            {
+                continue;
+            }
+            $output .= '// ' . $line . NEW_LINE;
+        }
+
+        return $output;
+    }
+
+    /**
+     * [SaveNexus Saves Nexus config files]
+     * @param boolean $fname [nexus config file name with path]
+     * @param [mixed]  $vars  [merged vars from config file and the $nexusConfig array]
+     */
+    function SaveNexus($fname = false, $vars)
+    {
+        $src = '<?php ' . NEW_LINE .
+       '/*' . NEW_LINE .
+       '* As a work of the United States government, this project is in the public domain within the United States.' . NEW_LINE .
+       '*/' . NEW_LINE .
+       '/*' . NEW_LINE .
+        TAB . 'General configuration ' . NEW_LINE .
+        TAB . 'Date: August 9, 2011' . NEW_LINE .
+        TAB . 'Central place to put org. chart config' . NEW_LINE .
+        TAB . 'This should be kept outside of web accessible directories' . NEW_LINE .
+        '*/' . NEW_LINE . NEW_LINE .
+        "// require '../../../config.php';" . NEW_LINE . NEW_LINE .
+        'namespace Orgchart;' . NEW_LINE .
+        "ini_set('display_errors', 0); // Set to 1 to display errors" . NEW_LINE . NEW_LINE .
+
+        'class Config' . NEW_LINE .
+        '{';
+        foreach ($vars as $var)
+        {
+            $src .= NEW_LINE . TAB . 'public ' . ($var['varType'] ? $var['varType'] . ' ' : $var[VAR_TYPE] . ' ') . '$' . ($var[VAR_NAME] ? $var[VAR_NAME] : $var['varName']) . ' = ' . ($var['varValue'] ? $var['varValue'] : $var[VAR_VALUE]) . ';' . ' ' .
+            formatComments(wordwrap(($var['varComment'] ? $var['varComment'] : ''), WORD_WRAP));
```

Comment:
These ternary operators need to check for isset, not just the value of the array at this index. (i.e. `isset($var['varName']) ?...` not `$var['varName']`)